### ISSUE TITLE
Add stalebot to help the marking of PRs

### ISSUE THREAD

**Issue by macgills** [commons-app/apps-android-commons#3583]
**Summary:** 

With the volume of PRs it is easy for some to fall through the cracks. With [stalebot](https://github.com/probot/stale) we could get a notification if there has been no activity for X number of days. Personally in the past I have used 7 days, issued a warning that the next stale marking will result in closing of the PR and then closed it after a total of 14 days inactivity

**Comment by misaochan** [comment]
I'm OK with using stalebot for PRs (probably not for issues). I guess a more suitable number of days of inactivity for our repo would be more like 28 (so warning after 21 days and closure after another 7?).

**Comment by macgills** [comment]
I don't think you can have it go 28-7 automatically. My flow was, stale 7 days>automatic comment and labelling stale> I comment which automatically removes Stale label> 7 days of inactivity reapplies stale label>I close

So I think we would want to choose 17 days then as the stale interval? Or a 3/4 stale strike policy

**Comment by Rishavgupta12345** [comment]
@nicolas-raoul I want to work on this issue please assign me

**Comment by nicolas-raoul** [comment]
As we lack reviewers, some reasonable PRs have been sleeping for much more than 14 days. I don't see it as a problem, we review PRs when we have time :-)
